corehq/apps/export/models/new.py

```diff
@@ -68,8 +72,9 @@ class ExportColumn(DocumentSchema):
     item = SchemaProperty(ExportItem)
     label = StringProperty()
     # Determines whether or not to show the column in the UI Config without clicking advanced
-    show = BooleanProperty(default=False)
+    is_advanced = BooleanProperty(default=False)
```

Comment:
We have certain Colin's that are considered advanced. Refers to button "Show Deleted Questions and Advanced properties". Let me know if you don't think the comment is clear above it